modules/dynamic-plug-in-api.adoc

```diff
@@ -367,7 +370,7 @@ selections
 
 |`\{string} [options.columnManagementID]` |(optional) A unique id
 used to persist and retrieve column management selections to and from
-user settings. Usually a 'group~verion~kind' string for a resource.
+user settings. Usually a `group~version~kind` string for a resource.
```

Comment:
There is another `group~version~kind` elsewhere, and I wasn't sure if the `~` was intentional or required formatting for the string, because it does render the "version" as a subscript. 🤷 If this was just a quirk of the upstream docs (and isn't a technical requirement to use `~`), it could alternatively be changed to our docs repo's [term glossary suggestion](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#groupversionkind-gvk) of (non-marked-up) group/version/kind (GVK).